autospec/avxjudge.py

```diff
@@ -0,0 +1,512 @@
+    avx2_instructions_ymm.append("vpminsq")
+    avx2_instructions_ymm.append("vpmuludq")
+    avx2_instructions_ymm.append("vpand")
+    avx2_instructions_ymm.append("vpmaxud")
+    avx2_instructions_ymm.append("vpminud")
+    avx2_instructions_ymm.append("vpmaxsd")
+    avx2_instructions_ymm.append("vpmaxsw")
+    avx2_instructions_ymm.append("vpminsd")
+    avx2_instructions_ymm.append("vpminsw")
+    avx2_instructions_ymm.append("vpand")
+    avx2_instructions_ymm.append("vpor")
+    avx2_instructions_ymm.append("vpmulld")
+
+
+    avx2_instructions.append("vfmadd132ss")
+    avx2_instructions.append("vfmadd213ss")
+    avx2_instructions.append("vfmadd231ss")
+    avx2_instructions.append("vfmadd132sd")
+    avx2_instructions.append("vfmadd231sd")
+    avx2_instructions.append("vfmadd213sd")
+
+    avx2_instructions.append("vfmsub132ss")
+    avx2_instructions.append("vfmsub213ss")
+    avx2_instructions.append("vfmsub231ss")
+    avx2_instructions.append("vfmsub132sd")
+    avx2_instructions.append("vfmsub231sd")
+    avx2_instructions.append("vfmsub213sd")
+
+    avx2_instructions.append("vfnmadd132ss")
+    avx2_instructions.append("vfnmadd213ss")
+    avx2_instructions.append("vfnmadd231ss")
+    avx2_instructions.append("vfnmadd132sd")
+    avx2_instructions.append("vfnmadd231sd")
+    avx2_instructions.append("vfnmadd213sd")
+
+    avx2_instructions.append("vfnmsub132ss")
+    avx2_instructions.append("vfnmsub213ss")
+    avx2_instructions.append("vfnmsub231ss")
+    avx2_instructions.append("vfnmsub132sd")
+    avx2_instructions.append("vfnmsub231sd")
+    avx2_instructions.append("vfnmsub213sd")
+
+    avx2_instructions_hv.append("vpclmulhqlqdq")
+    avx2_instructions_hv.append("vpclmullqhqdq")
+
+    avx2_instructions_hv.append("vfmadd132ps")
+    avx2_instructions_hv.append("vfmadd213ps")
+    avx2_instructions_hv.append("vfmadd231ps")
+    avx2_instructions_hv.append("vfmadd132pd")
+    avx2_instructions_hv.append("vfmadd231pd")
+    avx2_instructions_hv.append("vfmadd213pd")
+    avx2_instructions_hv.append("vfmsub132ps")
+    avx2_instructions_hv.append("vfmsub213ps")
+    avx2_instructions_hv.append("vfmsub231ps")
+    avx2_instructions_hv.append("vfmsub132pd")
+    avx2_instructions_hv.append("vfmsub231pd")
+    avx2_instructions_hv.append("vfmsub213pd")
+
+    avx2_instructions_hv.append("vfnmadd132ps")
+    avx2_instructions_hv.append("vfnmadd213ps")
+    avx2_instructions_hv.append("vfnmadd231ps")
+    avx2_instructions_hv.append("vfnmadd132pd")
+    avx2_instructions_hv.append("vfnmadd231pd")
+    avx2_instructions_hv.append("vfnmadd213pd")
+    avx2_instructions_hv.append("vfnmsub132ps")
+    avx2_instructions_hv.append("vfnmsub213ps")
+    avx2_instructions_hv.append("vfnmsub231ps")
+    avx2_instructions_hv.append("vfnmsub132pd")
+    avx2_instructions_hv.append("vfnmsub231pd")
+    avx2_instructions_hv.append("vfnmsub213pd")
+    avx2_instructions_hv.append("vdivpd")
+
+
+    return
```

Comment:
return not needed.

Also, if all this function is doing is setting the values for these instruction lists, this is a case where you could use a global and just set it to the literal list of values.

Reply:
In the context of this code, I agree that it makes more sense for these lists to be initialized in global scope (at the top of the source file), and no `init_ins()` call would then be needed.